ui/webpack.prod.js

```diff
@@ -0,0 +1,13 @@
+const webpack = require("webpack")
+const merge = require('webpack-merge');
+const UglifyJSPlugin = require('uglifyjs-webpack-plugin');
```

Comment:
Does this actually add any significant performance benefit when the files are being loaded locally? I suppose marginally, since the file still needs to be read from disk and interpreted, but my guess is it is very minor. 

At the same time, we fairly frequently ask users to give console output related to users.

If there's no real performance win here, I'd suggest this is not worth the added difficulty in debugging user issues, but I could possibly be persuaded otherwise.

Reply:
Good point. I'll take it out for now until I can get some real performance numbers. Then we can decide.

Reply:
@seanyesmunt once this is removed, go ahead and merge this into master.

Recommended merging practices are here: https://github.com/lbryio/lbry/wiki/Branching-and-Merging

I'd say I follow these about 80% of the time :

Reply:
@liamcardenas you should check out above link as well